src/mono/mono/mini/aot-compiler.c

```diff
@@ -5235,6 +5243,25 @@ add_full_aot_wrappers (MonoAotCompile *acfg)
 			add_method (acfg, mono_marshal_get_ptr_to_struct (klass));
 		}
 	}
+
+	/* unsafe accessor wrappers */
+	rows = table_info_get_rows (&acfg->image->tables [MONO_TABLE_METHOD]);
+	for (int i = 0; i < rows; ++i) {
+		ERROR_DECL (error);
+		token = MONO_TOKEN_METHOD_DEF | (i + 1);
+		method = mono_get_method_checked (acfg->image, token, NULL, NULL, error);
+		report_loader_error (acfg, error, TRUE, "Failed to load method token 0x%x due to %s\n", i, mono_error_get_message (error));
+
+		if (G_LIKELY (mono_method_metadata_has_header (method)))
```

Comment:
Shouldn't this check that the method has the 'extern' flag instead ?

Reply:
Is there an API doing this?

Reply:
There is no `extern` flag.  `extern` is an RVA column with a 0 in it. https://sharplab.io/#v2:C4LglgNgPgAgTARgLACgDEA7ArhCBDAIwgFMACAEzAGdCTUYBmU+UgYVIG9VSfmmYEANlLEAHsGIAnDMwAspAGIB7JQAoAlAG5UAXyA=